docs/demos/Nodes.story.tsx

```diff
@@ -1,18 +1,75 @@
 import React, { useState } from 'react';
-import { GraphCanvas, LayoutTypes } from '../../src';
-import { iconNodes, manyNodes, simpleEdges, simpleNodes, simpleNodesColors } from '../assets/demo';
+import { GraphCanvas, Icon, LayoutTypes, SphereWithIcon } from '../../src';
+import {
+  iconNodes,
+  manyNodes,
+  simpleEdges,
+  simpleNodes,
+  simpleNodesColors
+} from '../assets/demo';
 
 export default {
   title: 'Demos/Nodes',
   component: GraphCanvas
 };
 
-export const NoEdges = () => (
-  <GraphCanvas nodes={manyNodes} edges={[]} />
-);
+export const NoEdges = () => <GraphCanvas nodes={manyNodes} edges={[]} />;
 
 export const Icons = () => (
-  <GraphCanvas nodes={iconNodes} edges={simpleEdges} />
+  <GraphCanvas
+    nodes={iconNodes}
+    edges={simpleEdges}
+    renderNode={({ id, node, animated, nodeSize, opacity }) => (
+      <Icon
+        id={id}
+        image={node.icon || ''}
+        size={nodeSize + 8}
+        opacity={opacity}
+        animated={animated}
+      />
+    )}
+  />
+);
+
+export const Custom3DNode = () => (
+  <GraphCanvas
+    nodes={simpleNodes}
+    edges={simpleEdges}
+    cameraMode="rotate"
+    renderNode={({ node, active, nodeSize, theme, opacity }) => (
```

Comment:
I think `nodeTemplate` might be a better name for this.

Reply:
`renderX` is what I normally use for [renderProps](https://reactjs.org/docs/render-props.html), but I'm not sure if there's an accepted convention for it - most things I've seen usually have `render` or `renderer` in there somewhere